cpp/ql/test/library-tests/functions/functions/functions.cpp

```diff
@@ -30,4 +30,12 @@ class Table {
   bool insert(Name*);
 };
 
-
+class MyClass
+{
+public:
+	MyClass();
+	MyClass(int from);
+	MyClass(const MyClass &from);
+	MyClass(const MyClass &&from);
```

Comment:
Please either remove `const` or add another move constructor that doesn't have `const`.

Reply:
Done.